src/hooks/init.ts

```diff
@@ -33,8 +33,12 @@ export const hook: Hook<"init"> = async function (options) {
 
     const projectConfig = load();
 
-    if (!isOnVpn()) {
-        this.warn("Not on VPN. Some containers may not build properly.");
+    if (isWebProxyHostSet()) {
+        if (!isOnVpn()) {
+            this.warn("Not on VPN. Some containers may not build properly.");
+        }
+    } else {
+        this.warn("CH_PROXY_HOST env not set. Some containers may not build properly.");
     }
```

Comment:
Slight feature creep here - consider the case whereby the user hasn't got the  required environment variable set - is this the correct ux behaviour?

Reply:
If they are on the VPN and this var is not set, then something odd has happened. This is sort of a proxy for other bits of chproxyon not having run (maven, git...).

Reply:
as for the ux - is there someone to talk to about the correct ux message to show the user?